dts/bindings/spi/andestech.atcspi200.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2019, Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description:
+    This binding gives a representation of Andes ATCSPI200 SPI controller
+
+compatible: "andestech,atcspi200"
+
+include: spi-controller.yaml
+
+properties:
+
+    reg:
+      required: true
+
+    num-cs:
+      type: int
+      required: false
```

Comment:
IIRC `required` defaults to `false`, so there's no need to specify it if you don't want to make this field required.

Reply:
Oh! I got it.
Thank you for your help, I have fixed it.